code/modules/research/designs/nanotrasen.dm

```diff
@@ -31,3 +31,13 @@
 	build_path = /obj/item/weapon/gun/gravitywell
 	locked = 1
 	req_lock_access = list(access_rd)
+
+/datum/design/nuclear_gun
```

Comment:
I'm almost positive that giving these two designs the exact same path is going to cause a problem. Change the modkit's to nuclear_gun_modkit or something